<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gallery_Widget</class>
 <widget class="QWidget" name="Gallery_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>700</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labMove1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labMove2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labMove1</zorder>
  <zorder>labMove2</zorder>
  <zorder>btnBack</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
